sdk/cosmosdb/cosmos/test/public/functional/NonStreamingQueryPolicy.spec.ts

```diff
@@ -692,4 +698,221 @@ describe.skip("Full text search feature", async () => {
     assert(result.resources.length === 2);
     await database.container(containerName).delete();
   });
+
+  it("should execute a full text query targeting an individual partition via query plan using WHERE clause", async () => {
```

Comment:
Is `NonStreamingOrderBy flag = true` int the queryPlan for the queries where FullTextContains is used?